samples/csharp/getting-started/AdvancedExperiment_AutoML/README.md

```diff
@@ -0,0 +1,141 @@
+# Taxi Fare Prediction - Advanced AutoML Experiment
+
+## Automated Machine Learning
+
+Automated machine learning (AutoML) automates the end-to-end process of applying machine learning to real-world problems. Given a dataset, AutoML iterates over different data featurizations, machine learning algorithms, hyperparamters, etc. to select the best model based on training scores. 
```

Comment:
We'll hopefully be choosing on more criteria besides the training scores, like training speed, model size, prediction speed, memory, etc in the future...
```suggestion
Automated machine learning (AutoML) automates the end-to-end process of applying machine learning to real-world problems. Given a dataset, AutoML iterates over different data featurizations, machine learning algorithms, hyperparamters, etc. to select the best model for the dataset. 
```

Reply:
Ah, thanks! (Nit: since the sentence starts with 'Given a dataset', going to just to end sentence after 'model')